### PR TITLE
Have vote confirmed messages be a quick MB reaction rather than posting in the chat. (From Trello)

### PR DESCRIPTION
**Pull request by LW001**
Yeah so it shows f1 for 2.5s in chat now and then removes it again, thats literally it

https://trello.com/c/KbaNQq0J/

PS please look whether I didn't ruin it all :^)

### CODE REVIEW

**Comment by Dougley** [comment]
[Have vote confirmed messages be a quick MB reaction rather than posting in the chat.](https://trello.com/c/KbaNQq0J/64-have-vote-confirmed-messages-be-a-quick-mb-reaction-rather-than-posting-in-the-chat)

File: Commands/Commands/queue.js [hunk 1]
```diff
@@ -419,9 +419,10 @@ commands.registerVote = {
                       message: 'Feed-voted',
                       affected: doc.UvId
                     })
-                    bot.Channels.get(config.discord.feedChannel).sendMessage(`${user.mention}, your vote has been registered.`).then(c => {
-                      setTimeout(() => c.delete(), 5000)
-                    })
+                    doc.addReaction({
```
Comment (Dougley) [inline Commands/Commands/queue.js]: `doc` is the response from the database, not the message object.
Use `msg`.

**Review by Dougley — CHANGES_REQUESTED** [review]

File: Commands/Commands/queue.js (hunk 1 above)
Comment (LW001) [reply Commands/Commands/queue.js]: I am stupid.

**Comment by Arzte** [comment]
@LW001 FTFY

File: Commands/Commands/queue.js [hunk 2]
```diff
@@ -419,9 +419,10 @@ commands.registerVote = {
                       message: 'Feed-voted',
                       affected: doc.UvId
                     })
-                    bot.Channels.get(config.discord.feedChannel).sendMessage(`${user.mention}, your vote has been registered.`).then(c => {
-                      setTimeout(() => c.delete(), 5000)
-                    })
+                    msg.addReaction({
+                      id: '322758965547106305',
+                      name: 'f1'
+                    }).then(setTimeout(() => doc.removeReaction({id: '322758965547106305', name: 'f1'}), 2500))
```
Comment (Dougley) [inline Commands/Commands/queue.js]: You've forgot one

**Review by Dougley — CHANGES_REQUESTED** [review]

File: Commands/Commands/queue.js (hunk 2 above)
Comment (Arzte) [reply Commands/Commands/queue.js]: let me amend the commit and rebase, should be fine

**Review by Dougley — APPROVED** [review]

**Pull request merged**